app/src/main/java/org/dicekeys/app/fragments/ListDiceKeysFragment.kt

```diff
@@ -43,9 +37,18 @@ class ListDiceKeysFragment : AppFragment<ListDicekeysFragmentBinding>(R.layout.l
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        getNavigationResult<String>(ScanFragment.READ_DICEKEY)?.observe(viewLifecycleOwner) {
+            it?.let {
```

Comment:
I think the `it`s mean...

`facesReadJsonOrNull -> facesReadJsonOrNull?.let { facesReadJson ->`